modules/ossm-install-jaeger.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+// * serverless/installing-openshift-serverless.adoc
+
+[id="ossm-operator-install-jaeger_{context}"]
+= Installing the Jaeger Operator
```

Comment:
For standalone Jaeger, we need instructions on installing the Jaeger operator - as this may be the component they are wanting to use. However, as part of Service Mesh, this step should no longer be required with OSSM 1.0.2  and OCP 4.2.

Reply:
Breaking up this topic means I can reuse this module for standalone.  I hadn't realized that the dependencies were coming in 1.0.2 and not 1.1.